working-groups/user-interface/MeetingNotes/19-3-22.md

```diff
@@ -0,0 +1,36 @@
+# 19-03-2022 UI Working Group weekly meeting 
+
+## Attendees
+- Ananyaa Shrivastava
+- Gunjan Chhablani
+- Ishita Choudhary
+- Palak Sharma
+- Shloka Gupta
+
+
+## Agenda
+- Closing TSC Notice board project
+- ~Moving onto next project - FLINT-UI~
+
+## Discussion
+- Discuss GSoC projects
+- Second project: Building UI library
+    - A lot of components can be used in multiple places.
+    - Build UI based on that.
+    - Need to study the UI projects
+        - FLINT-UI, Community Website, FLINT Reporting Tool, FLINT.json
+    - Example: Header/Footer can be used as a component all across moja.global
+    - After studying, can find common things for each project.
+    - Work on FLINT-UI project.
+
+- [Retrospective for TSC WG Notice Board](https://miro.com/welcomeonboard/cE94aUZCa1h0VVBMTTlVWndCNUJqVkk5NVZlUVAyakFuUVREekJncEUxOHA1Z29XdGxnMVhXcFQ5TlJoU2pJa3wzMDc0NDU3MzYwMTUwNjUwNjE1?invite_link_id=281030745573)
+
+## Check in on previous action items
+- [ ] ~Sukriti to merge TSC WG PR.~
+- [x] Hold a closing note meeting for TSC WG noticeboard.
+    - [x] Discuss the project, what all can be improved, etc.
+- [ ] ~Start with RothC and Point from the next meeting.~
+
+## New Action items
+- [ ] Start with RothC and Point from the next meeting.
```

Comment:
Can you please replace this point with - "A focus group. Get people familiar with Vue, AntDesign, and Storybook".